src/renderers/art/ReactART.js

```diff
@@ -93,7 +93,7 @@ function injectAfter(parentNode, referenceNode, node) {
 
 // ContainerMixin for components that can hold ART nodes
 
-const ContainerMixin = assign({}, ReactMultiChild, {
+const ContainerMixin = assign({}, ReactMultiChild.prototype, {
 
```

Comment:
I understand but ART lives in its own repo. It's only here for running tests. (Yea I know it's confusing.) This means that if we had to change ART in a PR here, real ART will break with next version of React that contains this change. cc @spicyj @zpao